js/libs/core.js

```diff
@@ -354,6 +354,25 @@ Fliplet.FormBuilder = (function() {
           }
         }
 
+        if (this._componentName === 'flSlider') {
+          if (!this.step) {
+            data.step = 1;
+          }
+
+
+          if (!this.min) {
+            data.min = 0;
+          }
+
+          if (!this.max) {
+            data.max = 100;
+          }
+
+          if ((Number(data.max) - Number(data.min)) % Number(data.step) != 0) {
```

Comment:
Seems less messy if you just set `data.max = Number(data.max)` all before checking the values so that they are all numbers.

Reply:
done